recipes/simple-build-dgcode/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set version = "0.0.15" %}
+
+#NB: get sha256 from https://pypi.org/project/simple-build-dgcode/#files
+
+package:
+  name: simple-build-dgcode
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/simple-build-dgcode/simple-build-dgcode-{{ version }}.tar.gz
+  sha256: 1ec3efe27f2d1cb8f76c86b7daef73f99af15cd3dd1a67b1535329d25d0ca217
+
+build:
+  noarch: python
+  script: python -m pip install --no-deps -vv .
+  number: 0
+  entry_points:
+    - dgcode_newsimproject = simplebuild_dgcode._cli:newsimproject
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools-git-versioning
+    - setuptools
+  run:
+    - python >=3.8
+    - simple-build-system >=0.0.39
+
+test:
+  requires:
+    # Note: The actual tests are defined in run_test.sh
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

```suggestion
    - c-compiler
    - cxx-compiler
```

Let's try these to get rid of those hashes.

---

Thanks! I had absolutely no idea that packages in the test requirements would affect the hash of the package :-)